### DISCUSSION TITLE
Error in OpenRadioss Strain Output?

### DISCUSSION THREAD

**Discussion by SyazwanSamad** [OpenRadioss/OpenRadioss#663] [Q&A] [answered]
I'm developing my own failure model which is mostly based on strain.
After implementing the model, I noticed that at stress triaxiality = 0.333 (uniaxial tensile), the strain ratio calculated by OpenRadioss is 0.300. This is not correct where it should be 0.5. I have rechecked the simulation using LS-Dyna, and it did give me the value of 0.498 (~0.5). I'm not sure whether this is my mistake, or the solver's mistake. I'm sharing the one element model for your reference.
[SINGLE_ELEM.zip](https://github.com/OpenRadioss/OpenRadioss/files/9852394/SINGLE_ELEM.zip)

Thank you
Regards
Syazwan

**Comment by MarianBulla** [comment]
Hi, I will have a deeper look at it.
Will let you know.
Thanks and best regards,
Marian

**Comment by MarianBulla** [comment, marked as answer]
Can you retry it by setting the values in the property card (Ithick and Iplas) = 1, please ?
(It is the better choice for metals. Equals zero is metter for composites, but it also depends on measured results.)
Hope it helps you ?
Thanks and best regards
Marian

**Reply by SyazwanSamad** [reply]
Hi Marian,
After changing the settings, the results are now similar the theoretical value. Thank you! 

I'll make sure to use the /DEF* keyword to ensure this will never happened again. I'm currently typing the keyword manually, hence, have forgotten about the two critical options. 

Thank you again, 
Regards
Syazwan
